install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml

```diff
@@ -0,0 +1,45 @@
+SEM: {
```

Comment:
Could you add somewhere as a comment that the Support PC should be running xtadapter?